src/app.ts

```diff
@@ -0,0 +1,90 @@
+import { Probot, Context } from 'probot';
+
+interface PullRequestInfo {
+  owner: string
+  repo: string
+  pull_number: number
+}
+
+const getPRInfo = async (context: Context, {
+  owner, repo, pull_number
+}: PullRequestInfo) => {
+  const {
+      data: {
+        commits,
+        head
+      }
+    } = await context.octokit.pulls.get({
+      owner, repo, pull_number
+    });
+  
+  return { prNumCommits: commits, prHeadSha: head.sha, prBranchName: head.ref }
```

Comment:
nit: I don't think we need to prefix these properties with `pr` since the function is called `getPRInfo` so it should be expected that the info these properties have are for the PR.

This'll let you use shorthand properties in later code :)